scripts/build-docker

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ARCH=$(go env GOARCH)
```

Comment:
If the docker images have to be built on different machines, then they need to pull the binaries from a master set of binaries. Rebuilding the binaries on different platforms is an invitation for generating images with binaries don't match the set of signed binaries.